testing/test_aux.sh

```diff
@@ -160,7 +160,7 @@ echo Redacted docs diff | tee -a $TEST_RESULTS
     | tee -a $TEST_RESULTS
 
 # Make sure there's no file pollution from the test run.
-git status --porcelain | tee -a $TEST_RESULTS
+git status --porcelain -u no | tee -a $TEST_RESULTS
```

Comment:
The entire purpose of this was to make sure there was no pollution side-effects... what is it specifically that you're trying to ignore?

Reply:
It's always writing to misc/docker_images.txt which I ignored explicitly in .gitignore. I also don't want this file to be changed accidentally.

Reply:
Ok -- so -- my question is likely then why are you ignoring that file?   How do you know which images to download?

Reply:
I have committed the file then ignored it. The file is technically untracked under this condition so it won't show up under git status -u no.